javascript/javascript2/week3/fastest-presser.js

```diff
@@ -0,0 +1,72 @@
+/*
+When the button is clicked, get the value of the input.
+This value will be the amount of time the game should run.
+*/
+
+const gameButton = document.querySelector("#start"); //start game button
+const input = document.querySelector("input");
+let sCount = 0;
+let lCount = 0;
+
+const resultTag = document.getElementById("game-result"); //game result
+resultTag.innerHTML = "";
+const gameResult = () => {
+  if (sCount == 0 && lCount == 0) {
+    resultTag.innerHTML = "No action!!!";
+  }
+  if (sCount == lCount && sCount > 0 && lCount > 0) {
+    resultTag.innerHTML = "Game Draw!!!";
+  }
+  if (sCount > lCount) {
+    resultTag.innerHTML = "S is the winer !!!";
+
+    const confettiSettings = { target: "s-canvas", size: 2 };
+    const confetti = new ConfettiGenerator(confettiSettings);
+    confetti.render();
+  }
+  if (lCount > sCount) {
+    resultTag.innerHTML = "L is the winner";
+    const confettiSettings = { target: "l-canvas", size: 2 };
+    const confetti = new ConfettiGenerator(confettiSettings);
+    confetti.render();
+  }
+};
+
+gameButton.addEventListener("click", gameStart);
+function gameStart() {
+  let gameTime = input.value;
+  if (gameTime == "") {
```

Comment:
Is good practice to use `===` instead `==` , This comment will also apply on line 53, 57